sdk/personalizer/Azure.AI.Personalizer/src/Generated/PersonalizerV1Preview1Client.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.AI.Personalizer.Models;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.Personalizer
+{
+    /// <summary> The PersonalizerV1Preview1 service client. </summary>
+    public partial class PersonalizerV1Preview1Client
```

Comment:
Could we have a better name for this, one without all the version info?

Reply:
I will change the name and regenerate the SDK